src/java/org/apache/cassandra/cql3/restrictions/SimpleRestriction.java

```diff
@@ -403,7 +402,7 @@ else if (isIN())
 
     private static ByteBuffer multiInputOperatorValues(ColumnMetadata column, List<ByteBuffer> values)
     {
-        return ListType.getInstance(column.type, false).pack(values, ByteBufferAccessor.instance);
+        return ListType.getInstance(column.type, false).pack(values);
```

Comment:
when i did the last rebase this change got added else the history got too hard to work with; so rolling back